music_assistant/controllers/streams.py

```diff
@@ -609,10 +617,12 @@ async def serve_queue_flow_stream(self, request: web.Request) -> web.Response:
             reason="OK",
             headers=headers,
         )
-        http_profile_value = await self.mass.config.get_player_config_value(
-            queue_id, CONF_HTTP_PROFILE
+        http_profile = (
+            await self.mass.config.get_player_config_value(
+                queue_id, CONF_HTTP_PROFILE, return_type=str
+            )
+            or "default"
```

Comment:
we should maybe also add a default/fallback to the get_player_config_value to make statements like these easier

Reply:
Added that - if you update this branch you can use it

Reply:
default parameter now used throughout